src/main/java/Duke.java

```diff
@@ -1,10 +1,135 @@
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.Scanner;
+
 public class Duke {
+
+    private static final int LINE_WIDTH = 60;
+
+    //to draw a straight line
+    private static void printHorizontalLine() {
+        System.out.println("_".repeat(LINE_WIDTH));
+    }
+
+    public static String[] taskLister(String[] todos) {
+
+        int todoCount = 0;
+        String[] result = new String[100];
+        for (String todo : todos) {
+            //System.out.println(word);
```

Comment:
Remove unnecessary println comments originally used for debugging, helps to make code more tidy